src/lookup.rs

```diff
@@ -50,6 +52,31 @@ pub(super) struct ChannelDelta {
 	pub(super) requires_reminder: bool,
 }
 
+pub(super) struct NodeDelta {
+	/// The most recently received, but new-to-the-client, node details
+	pub(super) latest_details_after_seen: Option<NodeDetails>,
+
+	/// Between the last details prior to the `seen` value and now, including any potential
```

Comment:
Is this the same as saying `last_details_before_seen`? If so, can we say that instead?